### DISCUSSION TITLE
Replacement for `metaTitle`?

### DISCUSSION THREAD

**Discussion by chris-dura** [vuepress/core#451] [Q&A] [answered]
There is no info in the migration guide on the default meta title: https://vuepress.vuejs.org/guide/frontmatter.html#metatitle

What is the suggestion when you want to override how the `document.title` is displayed?

**Comment by Mister-Hope** [comment, marked as answer]
You should use resolvers. It's obviously more powerful.

https://v2.vuepress.vuejs.org/reference/client-api.html#resolvers
